### ISSUE TITLE
Error Importing React.Node type - `bpfrpt_proptype_Node is not defined`

### ISSUE THREAD

**Issue by SpainTrain** [brigand/babel-plugin-flow-react-proptypes#171]
Opening just for tracking based on thread starting at https://github.com/brigand/babel-plugin-flow-react-proptypes/issues/136#issuecomment-360248347

From that thread:

> Ah, well there are two issues here. The import name was wrong between 14.0.0 and 14.0.3, but the current issue is that we don't detect the named import of Node from 'react', so it ends up producing this:

```js
import React from 'react';


const C = props => React.createElement('div', null);
C.propTypes = {
  children: function () {
    return (typeof bpfrpt_proptype_Node === 'function' ? bpfrpt_proptype_Node.isRequired ? bpfrpt_proptype_Node.isRequired : bpfrpt_proptype_Node : PropTypes.shape(bpfrpt_proptype_Node).isRequired).apply(this, arguments);
  }
};
import { bpfrpt_proptype_Node } from 'react';
import PropTypes from 'prop-types';
```

> In 14.0.3 it should at least not error (I think), but the generated propTypes are wrong.

**Comment by patrikholcak** [comment]
Any updates on this? Here’s a workaround until this gets fixed:

```js
import * as React from 'react'

type Props = {
  children: React.Node
}

class App extends React.Component<Props> { … }
```

**Comment by brigand** [comment]
Sorry for the delay on this. Published as 21.0.0.
